lib/http1.c

```diff
@@ -153,7 +153,7 @@ static void entity_read_send_error(struct st_h2o_http1_conn_t *conn, int status,
     set_timeout(conn, NULL, NULL);
     h2o_socket_read_stop(conn->sock);
     conn->req.http1_is_persistent = 0;
-    h2o_send_error(&conn->req, status, reason, body, H2O_SEND_ERROR_HTTP1_CLOSE_CONNECTION);
+    h2o_send_error_generic(&conn->req, status, reason, body, H2O_SEND_ERROR_HTTP1_CLOSE_CONNECTION);
```

Comment:
Should we better invoke `h2o_send_error_XXX` in the callers of `entity_read_send_error` to collect the error codes?